tdvt/tdvt/tabquery.py

```diff
@@ -38,14 +37,15 @@ def get_max_process_level_of_parallelization(desired_threads):
         return 1
     return desired_threads
 
+
 def build_tabquery_command_line(work):
     try:
         sys.path.insert(0, get_extensions_dir())
         from extend_tabquery import TabqueryCommandLineExtension
         sys.path.pop(0)
         tb = TabqueryCommandLineExtension()
         logging.debug("Imported extension extend_tabquery")
-    except:
+    except ImportError:
```

Comment:
Will this more specific catch be an issue, or is it always this type?

Reply:
It would only be this type of error, thrown on 44.